resources/views/layouts/header.blade.php

```diff
@@ -35,30 +35,21 @@
         restarters.analyticsCookieEnabled = (typeof gdprCurrentCookiesSelection !== 'undefined' && gdprCurrentCookiesSelection['analytics']);
     </script>
 
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    @if( !empty(env('GOOGLE_ANALYTICS_TRACKING_ID')) )
-        <script async src="https://www.googletagmanager.com/gtag/js?id={{ env('GOOGLE_ANALYTICS_TRACKING_ID') }}"></script>
-        <script>
-        if (restarters.analyticsCookieEnabled) {
```

Comment:
Matomo say that you can get rid of the cookie banner:

https://matomo.org/cookie-consent-banners/

From memory it needs some configuration on the Matomo side for that.

However we might well need it for other purposes, and the Restarters demographic is very sensitive to this stuff.  There is a reasonable chance that the law will change to scrap cookie banners, but that's not certain yet.  

On Freegle, the absence of a cookie banner - which I'd worked hard to avoid needing - triggered concern from some advertisers, amusingly.

Reply:
Great, let's chat through Matomo's privacy-preserving cookieless option then, and make sure we've got that set up.  Assuming that's all fine we don't need that flag for checking opt-in to analytics.